src/helpers/lodash_alternatives.ts

```diff
@@ -0,0 +1,54 @@
+import { UnifiedIsoWeek } from './date-cache';
+
```

Comment:
I think that all the functions in this file deserve some documentation. If a function works exactly as in `lodash`, it might make sense to add a link to the lodash documentation of the corresponding function. If not, please document how it exactly works and maybe also point out the difference to lodash.